app/views/devise/passwords/new.html.erb

```diff
@@ -1,18 +1,24 @@
-<h2>Forgot your password?</h2>
+<div class="sign-up-wrapper" style="background-image: url(https://res.cloudinary.com/dvallavgs/image/upload/v1584451197/boy-child-clouds-kid-346796_t14hgc.jpg)">
+
 
 <%= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
   <%= f.error_notification %>
 
-  <div class="form-inputs">
+  <div class="container">
+    <div class="row black_bgr">
+      <div class="col-xs-12 col-sm-8 offset-sm-2">
+       <h2>Forgot your password?</h2>
+   <div class="form-inputs">
     <%= f.input :email,
                 required: true,
                 autofocus: true,
                 input_html: { autocomplete: "email" } %>
```

Comment:
what are these lines doing? autofocus, input_html etc @Max51555

Reply:
these are the default values added by 'devise" to send you a mail when you click on the btn 'forgot my e-mail'
not sure about the autofocus

Reply:
ah ok nice. thanks!